staging/src/k8s.io/component-base/metrics/testutil/metrics.go

```diff
@@ -38,6 +39,22 @@ var (
 	QuantileLabel model.LabelName = model.QuantileLabel
 )
 
+// Sample is model.Sample.
+type Sample model.Sample
+
+// NewSample returns new Sample which is initialized.
+func NewSample(kv map[string]string, value float64, time int64) *Sample {
```

Comment:
By checking callers of this function, `time` is always 0.
Do you have a plan to use this `time` for the other value?
If not, it is nice to remove it from this function for easy code.

Reply:
done